library/widget/widget_interactive.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) E.W.Ayers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+
+Author: E.W.Ayers
+-/
+import .basic .tactic_component .interactive_expr
+open widget
+/-- General purpose tactic for use with the widget system, setting the state to a component will render that component,
+so you can use this to show arbitrary components.
+The plan is that this will eventually replace `tactic`.
```

Comment:
This sounds potentially contentious.

Since this file is not used anywhere (except in the examples), maybe we could leave it out for now?

Reply:
yeah that's an old comment

Reply:
actually I was just using this for some tests so I'm refactoring it away